components/browser/engine-gecko-beta/src/test/java/mozilla/components/browser/engine/gecko/selection/GeckoSelectionActionDelegateTest.kt

```diff
@@ -0,0 +1,31 @@
+package mozilla.components.browser.engine.gecko.selection
+
+import android.app.Activity
+import android.app.Application
+import android.app.Service
+import mozilla.components.concept.engine.selection.SelectionActionDelegate
+import mozilla.components.support.test.mock
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Test
+
+class GeckoSelectionActionDelegateTest {
```

Comment:
We should add some tests here e.g. to make sure `getAllActions` contains the actions of the custom delegate.